website/docs/components/page-header/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,237 @@
+
+Representing metadata with a set of key/value pairs is common in HashiCorp products and can be useful when communicating relational information between products, versioning, and other structured data.
+
+![Custom metadata key/value pairs](/assets/components/page-header/page-header-custom-metadata.png =600x*)
+
+!!! Dont
+
+We recommend not exceeding more than four key/value pairs, this can result in unnecessarily complex content with the Page Header. Instead, move this content into the page.
+
+![Complex metadata dont](/assets/components/page-header/page-header-custom-metadata-dont.png)
+
+!!!
+
+## Actions
+
+Use [Buttons](/components/button) and [Dropdowns](/components/dropdown) in the Page Header to highlight functions and actions that impact the page as a whole. Examples of this include:
+
+- Creating new objects that are listed within the page
+- Surfacing management details like connecting to an API
+- Deleting or deactivating an object
+- Pausing or refreshing a service pertaining to the page
+
+![Page Header Actions](/assets/components/page-header/page-header-actions.png =853x*)
+
+### Primary button
+
+Use a `primary` [Button](/components/button) in the Page Header to highlight the most important flow or action on the page. Examples of this include:
+
+- A flow that creates an object that is related to the page, e.g., creating a new cluster or adding a new user.
+- Launching or deploying a specific product instance.
+
+### Secondary button
+
+Use a `secondary` [Button](/components/button) in the Page Header when highlighting actions that are less important than the primary action, such as editing the content on a page, managing access, etc.
+
+### Tertiary button
+
+Use a `tertiary` [Button](/components/button) when highlighting low-priority actions like linking to external documentation.
+
+![Page Header Actions Tertiary Button](/assets/components/page-header/page-header-actions-tertiary-button.png =700x*)
+
+### Dropdowns
+
+Use a [Dropdown](/components/dropdown) in the Page Header to combine multiple secondary functions, tasks, or elements that assist the user in managing the information or objects on the page. Examples of this include:
+
+- Copying API credentials to manage the object remotely
+- Editing or changing settings
+- Deleting or deactivating an object
+- Linking to documentation
+
+When used in these scenarios and paired with a `primary` button, use a `secondary` [Dropdown](/components/dropdown).
+
+!!! Do
+
+Use a primary [Dropdown](/components/dropdown) when the same action can be done in multiple different contexts. For example, taking the same action through a different interface or path; GUI, CLI, or API call.
+
+![Page Header primary dropdown](/assets/components/page-header/page-header-actions-do-primary-dropdown.png =853x*)
+
+!!!
+
+!!! Dont
+
+Don't use a `primary` [Dropdown](/components/dropdown) when combining different, unrelated actions. Instead, use the `secondary` color.
```

Comment:
> Instead, use the `secondary` color.
Should this be "dropdown" instead of "color"?

Reply:
Color refers to the property name, but I'll admit this doesn't read well. I'll clean it up a bit.